package.json

```diff
@@ -75,7 +76,12 @@
   },
   "prettier": {
     "arrowParens": "avoid",
-    "singleQuote": true
+    "singleQuote": true,
+    "importOrder": [
+      "^.\\/components\\/patterns\\/(.*)Components$",
```

Comment:
This regular expression ensures the imports for legacy components in the pattern library router are placed in their own group (as they are now), to ease eventual removal.

Once those are removed we can also remove this regexp.

Reply:
I'm adding removal of this to the list of small tasks for version 6: https://github.com/hypothesis/frontend-shared/issues/876